### ISSUE TITLE
Chained Multioutput Regression

### ISSUE THREAD

**Issue by monasabeti** [lsadouk/traffic_flow_code#3]
@namdre
i want to predict the the traffic flow by chained multioutput regression.
this is some part of my dataset:
![Capture](https://user-images.githubusercontent.com/43193011/149792900-78dc1c48-6672-43ae-9970-a28ebff632c3.PNG)
and i want to fit this dataset with the chained model but the result is not the same with the real data. how can i get a good result with high accuracy?
this is the code:
import pandas as pd
from sklearn.model_selection import cross_val_score
from sklearn.model_selection import RepeatedKFold
from numpy import mean
from numpy import std
from numpy import absolute
from sklearn.linear_model import LinearRegression
from sklearn.multioutput import RegressorChain
df = pd.read_csv('C:/Users/mona_/Sumo/stuttgart/training_dataset_chainig svr.xml')  # Importing the dataset
predictor=['input']
target_column=list(set(list(df.columns))-set(predictor))
df[predictor]= df[predictor]/df[predictor].max()
X = df[predictor].values
y = df[target_column].values
print(X.shape,y.shape)
model = LinearRegression()
wrapper = RegressorChain(model)
wrapper.fit(X, y)
row = [50]
yhat = wrapper.predict([row])
print('Predicted: %s' % yhat[0])
cv = RepeatedKFold(n_splits=10, n_repeats=3, random_state=1)
n_scores = cross_val_score(wrapper, X, y, scoring='neg_mean_absolute_error', cv=cv, n_jobs=-1, error_score='raise')
n_scores = absolute(n_scores)
print('MAE: %.3f (%.3f)' % (mean(n_scores), std(n_scores)))

and this is the result which is totally difference with the real data:
(1121, 1) (1121, 52)
Predicted: [4004.63399419  108.24223195  204.36019764  521.38831796  363.99825914
  391.82034793  328.24810424  342.06022474 1010.75101084  548.1733209
   75.39983325   27.54817292  407.6075746   156.50341349  136.74011876
    4.57705725 2297.46884076  625.14208371 3393.53745567  133.99415362
 2130.68485543  179.06254265  109.46837369  478.36685216 4072.44892581
  109.78732487  107.31126775  154.12125468 2093.94051794 1479.08705117
  249.18216898 3759.35603607 3471.29234906  309.41509741 2332.57857805
 3127.30731732  193.90058842  107.78288812  223.38752747  122.18555694
  396.29544121  361.31444637 1138.14966465 1018.65328421  146.63184061
 1725.49841208  487.87607118 2717.0058788  1876.45770725  122.36426857
   71.66776751  112.73573187]
MAE: 1.948 (0.102)
